### ISSUE TITLE
How to login with the account having 2-factor authentication turned on?

### ISSUE THREAD

**Issue by santosh** [instaloader/instaloader#200]
**Describe the bug**
400 Bad Request when trying to login, account having a 2-step authentication turned on.

**To Reproduce**
* Turn 2-step authentication
* Login from the terminal

**Expected behavior**
The application should log me in.

**Error messages and tracebacks**
```bash
$ instaloader --login=sntsh.k
Session file does not exist yet - Logging in.
Enter Instagram password for sntsh.k: 
Fatal error: Login error: 400 Bad Request
```
**Instaloader version**
4.1.1

I still receive the OTP on my phone.

**Comment by aandergr** [comment]
Thanks for this suggestion. Login with 2FA is not implemented yet in Instaloader.

**Comment by aandergr** [comment]
It's implemented in 4.2rc2. Upgrade with

```
pip3 install --pre --upgrade instaloader
```
